packages/server/README.md

```diff
@@ -17,72 +17,379 @@ All the provided endpoints are `Content-Type: application/json`
 
 #### 1. Users
 
-##### 1.1. Create
+##### 1.1. Check existing subscription details
```

Comment:
I think we should list these separately: create and get

Reply:
Ah yes I removed the create endpoint that is used from anj.
I will move Create endpoint to the end and name it "Create user without verifying"

Reply:
added at 1.11